docs/UserGuide.md

```diff
@@ -372,3 +379,16 @@ Action | Format, Examples
 **Clear** | `clear` then `confirm`
 **Export** | `export FILENAME`<br> e.g., `export exported_email`
 **Assign** | `assign STUDY_GROUP [STUDY_GROUP …]`<br> e.g., `assign P90-Control P90-Experiment`
+
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Misc. - Valid email field format:**<br>
+
+1. The local-part should **only contain alphanumeric characters** and these special characters, excluding the parentheses, (+_.-). The local-part may not start or end with any special characters.
+2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods.
+   The domain name must:
```

Comment:
can add then u must start with a domain label and not a period. like u cant do @.com
cos u say separated by periods so some might think u can start with a period lol

Reply:
Hm it said must start with alphanumeric tho. Plus it's the same as the email format error message so they might report an inconsistency there